packages/story-editor/src/components/floatingMenu/elements/textAlign.js

```diff
@@ -101,7 +102,10 @@ function TextAlign() {
   const [offsetLeft, setOffsetLeft] = useState(0);
 
   // Record left position of this button in the parent design menu
-  useEffect(() => setOffsetLeft(buttonRef.current.offsetLeft + OFFSET_X), []);
+  useEffect(
+    () => setOffsetLeft(buttonRef.current.parentNode.offsetLeft + OFFSET_X),
```

Comment:
Using `parentNode` here now as the button is wrapped into a tooltip. An alternative would be creating a new wrapper around the tooltip as well and using a `ref` for that, however, this might need some adjustment anyway related to #10872 cc @barklund